docs/CLI Commands.md

```diff
@@ -33,22 +29,30 @@ yarn haul start --eager # Prebuild bundle for all platforms
 yarn haul start --eager ios,windows # Prebuild bundle for iOS and Windows only
 ```
 
-### `--dev <true|false>`
+#### `--dev <true|false>`
 
 Whether to build in development mode, which sets the global `__DEV__` variable to `true`, and `process.env.NODE_ENV` to `development`. `true` by default.
 
-### `--minify <true|false>`
+#### `--minify <true|false>`
 
 Whether to minify the bundle while serving. This will make the bundling process slow due to extra minification step, and is `false` by default.
 
-### `--port [number]`
+#### `--port [number]`
 
 Port on which the server should run. defaults to `8081`.
 
-### `--config [path]`
+#### `--assets-dest <string>`
```

Comment:
I notice `--assets-dest` is described below as being where assets are stored for the build output, is this intended to be named something related to temp files instead?